src/Sql/Sql/ChangeLog.md

```diff
@@ -24,6 +24,7 @@
 * Fix missing examples for Set-AzSqlDatabaseSecondary cmdlet
 * Fix set Vulnerability Assessment recurring scans without providing any email addresses
 * Fix a small typo in a warining message.
+* Add failover database and elastic pool new cmdlets.
```

Comment:
@emgu-ms this line you're adding should go under `## Upcoming Release` -- I believe this happened due to merging issues after the latest release